.eslintrc.json

```diff
@@ -10,6 +10,9 @@
   "rules": {
     "no-multi-spaces": "off",
     "require-jsdoc": "off",
-    "valid-jsdoc": "off"
+    "valid-jsdoc": "off",
+    "no-unused-vars": ["warn",  { "vars": "local"}],
+    "no-prototype-builtins": "off"
```

Comment:
It's not obvious to me why you need to turn this off. Looking at the documentation, seems like it should be avoidable? https://eslint.org/docs/rules/no-prototype-builtins